src/static/index.ts

```diff
@@ -2,9 +2,11 @@
 //
 // Use of this source code is governed by the LICENSE file in this repository.
 
-import ClipboardJS from 'clipboard';
-import { App, Config, Elm, Flags, Theme } from '../elm/Main';
+// import types
+import { App, Config, Flags, Theme } from './index.d';
```

Comment:
Separated out types and moved type definition file in same folder. Maybe personal preference, but since this is the only TS occurrence in the project, I prefer the files close to each other.